docs/internal_api.yml

```diff
@@ -38,7 +38,76 @@ paths:
             description: Unexpected error.
             schema:
               $ref: '#/definitions/Error'
+ /tenant/{tenant_id}/limits/max_device_count:
```

Comment:
Also, I don't think there's any other internal endpoint that takes `tenant ID` explicitly. I suppose they all take `Authorization` header with a proper tenant token (i.e. token with `mender.tenant` claim).